jetty-openid/src/main/config/etc/jetty-openid.xml

```diff
@@ -1,12 +1,24 @@
 <?xml version="1.0"?>
 <!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Get id="ThreadPool" name="threadPool"/>
+  <New id="HttpClient" class="org.eclipse.jetty.client.HttpClient">
+    <Arg>
+      <New class="org.eclipse.jetty.util.ssl.SslContextFactory$Client">
+        <Set name="trustAll" type="boolean">false</Set>
+        <Set name="endpointIdentificationAlgorithm">https</Set>
```

Comment:
I would also make this a property. I don't now of any other algorithm though besides this one, so maybe that's not needed.